### PR TITLE
Adds base docker setup for local mopidy

### PR DESCRIPTION
**Pull request by atothewest**
First pass at satisfying issue #101 

If someone other than me could also pull this down to validate that it works correctly that would be great. Don't remove the "Do not merge" label until at least one other person has checked.

TODO:

- [x] Remove mopidy dependency in docker-compose - dockerised mopidy should be optional at this point.
- [x] Add documentation around process
- [x] Secondary confirmation that. It. Works.

### CODE REVIEW

File: mopidy/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+FROM debian:stretch-slim
+
```
Comment (whomwah) [inline mopidy/Dockerfile]: Is there a `debian:buster-slim` ?

File: mopidy/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        dumb-init \
+        gcc \
+        gnupg \
+        gstreamer1.0-alsa \
+        gstreamer1.0-plugins-bad \
+        python-crypto \
+        python-pykka \
+ && curl -L https://apt.mopidy.com/mopidy.gpg | apt-key add - \
+ && curl -L https://apt.mopidy.com/mopidy.list -o /etc/apt/sources.list.d/mopidy.list \
+ && apt-get update \
```
Comment (whomwah) [inline mopidy/Dockerfile]: Be interesting to see what this adds. The Pi version for instance adds:
```
pi@mopidy-staging:~ $ cat /etc/apt/sources.list.d/mopidy.list
# Mopidy APT archive
# Built on Debian 10 (buster)
deb http://apt.mopidy.com/ buster main contrib non-free
deb-src http://apt.mopidy.com/ buster main contrib non-free
```
hence the `buster` comment

File: mopidy/config/mopidy.conf [hunk 3]
```diff
@@ -0,0 +1,50 @@
+[core]
+cache_dir = /var/cache/mopidy
+config_dir = /etc/mopidy
+data_dir = /var/lib/mopidy
+
+[audio]
+# Only needed if running Icecast2
+# output = tee name=t ! queue ! audioresample ! autoaudiosink t. ! queue ! lamemp3enc ! shout2send mount=radio ip=127.0.0.1 port=8000 password=hackme
+mixer_volume = 40
```
Comment (whomwah) [inline mopidy/config/mopidy.conf]: you can ditch the two commented out lines

File: mopidy/config/mopidy.conf [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+port = 6680
+static_dir =
+
+[spotify]
+enabled = true
+username = 6kcscm6h0w888ujgcsnopkl3q
+password = xfvZNQjWVALAymf4FTAx
+client_id = 8a224b3c-b31c-4225-b881-3ec15bbea84f
+client_secret = URtYxUyqZLzAw2dPptkPeAvqa8d_NryacQJnutrj9Gc=
+private_session = true
```
Comment (whomwah) [inline mopidy/config/mopidy.conf]: @atothewest  Did you mean to add creds ?

Comment (atothewest) [reply mopidy/config/mopidy.conf]: D'oh 😂

File: mopidy/Dockerfile (hunk 1 above)
Comment (atothewest) [reply mopidy/Dockerfile]: 👍 Have updated to buster and all still working. Latest and greatest.

**Pull request merged**